### DISCUSSION TITLE
Jitendex entry for どろどろ looks weird

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#24] [Errors and Corrections]
Is this how it's supposed to look (the adverb part)?

![image](https://github.com/stephenmk/Jitendex/assets/146892521/1b81a703-72db-41a9-9356-3c186500b00d)

**Comment by stephenmk** [comment]
I think you have "compact tags" enabled in the "Popup Appearance" section of your settings. I suggest turning it off, since that feature was intended to solve a problem with the old version of jmdict for yomichan.

**Reply by stephenmk** [reply]
I also fixed the placement of that example sentence. Should be under "thick; viscous" rather than "rumbling."
